src/test/java/de/retest/web/it/TestHealerCssSelectorIT.java

```diff
@@ -0,0 +1,113 @@
+package de.retest.web.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import java.net.MalformedURLException;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import de.retest.web.AttributesUtil;
+import de.retest.web.selenium.AutocheckingRecheckDriver;
+
+class TestHealerCssSelectorIT {
+
+	private static WebDriver driver;
+
+	@BeforeAll
+	static void setup() throws MalformedURLException {
+		final ChromeOptions opts = new ChromeOptions();
+		opts.addArguments(
+				// Enable headless mode for faster execution.
+				"--headless",
+				// Use Chrome in container-based Travis CI enviroment (see https://docs.travis-ci.com/user/chrome#Sandboxing).
+				"--no-sandbox",
+				// Fix window size for stable results.
+				"--window-size=1200,800" );
+		//		final String page = "TestHealerCssSelectorIT.html";
+		final String page = "TestHealerCssSelectorIT-changed.html";
+		driver = new AutocheckingRecheckDriver( new ChromeDriver( opts ) );
+		driver.navigate().to( TestHealerCssSelectorIT.class.getResource( page ).toExternalForm() );

```

Comment:
You can obtain a pre-configured `ChromeDriver` from `WebDriverFactory`:

```java
driver = new AutocheckingRecheckDriver( WebDriverFactory.driver( CHROME ) );
```

I would also init `page` after`driver` to reduce the scope.